models/database.go

```diff
@@ -695,6 +695,22 @@ var databaseSchema = [][]string{
 		`ALTER TABLE percona_sso_details
 			ADD COLUMN pmm_server_name VARCHAR`,
 	},
+	60: {
+		`ALTER TABLE percona_sso_details
+			RENAME COLUMN client_id TO pmm_managed_client_id`,
+	},
+	61: {
+		`ALTER TABLE percona_sso_details
+			RENAME COLUMN client_secret TO pmm_managed_client_secret`,
+	},
+	62: {
+		`ALTER TABLE percona_sso_details
+			ADD COLUMN grafana_client_id VARCHAR NOT NULL`,
+	},
+	63: {
+		`ALTER TABLE percona_sso_details
+			ALTER COLUMN organization_id SET NOT NULL`,
+	},
```

Comment:
added this new migration because we discovered a bug here https://github.com/percona/pmm-managed/pull/1050/files#diff-046eff8c6fd93bc9a6ace012834425228856532b375e8b7e218fea20f284a88dR131  where we were setting the organization_id field to null

when organization_id was first introduced the old flow of connect PMM had already been shipped to customers and we didn't want to break things for them so we didn't add the NOT NULL constraint to organization_id  but now since we are going to ship the new connect flow which is not going to be backwards compatible and we expect to break things i guess it wouldn't hurt to add this migration. Correct me if I am wrong @BupycHuk